Package.swift

```diff
@@ -15,7 +15,11 @@ let package = Package(
     ],
     targets: [
         .target(
-            name: "ViewInspector", dependencies: []),
+            name: "ViewInspector", 
+            dependencies: [],
+            swiftSettings: [
+                .enableExperimentalFeature("StrictConcurrency")
```

Comment:
We may not need to declare it at the package level - if you open the project through `ViewInspector.xcodeproj` instead of `Package.swift`, that should be possible to enable this setting internally in the project